frontend/src/components/LikeButtonComment/LikeButtonComment.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../styles/prelude.scss";
+
+.like-component-comment{
+    display: flex;
+    flex-flow: row nowrap;
+    font-size: 12px;
+    position: relative;
+    right: -150px;
```

Comment:
I'm iffy about doing it this way...what exactly are you trying to do with this? Maybe I can direct you towards something else that would be better?

Reply:
move the like button to desired position for comment, without the nudge it's right next to the date

Reply:
Would you mind providing me with a screenshot of what you mean? I just want to make sure I'm understanding the right thing to give you the right advice :)

Reply:
![image](https://user-images.githubusercontent.com/45129406/77663220-362b5680-6f53-11ea-9e23-fb6a351a1e63.png)
the relative positioning puts the like button in the bottom right corner of its parent element, for both picture and comment